Index: src/Student.java
===================================================================
diff --git a/src/Student.java b/src/Student.java
new file mode 100644
--- /dev/null	
+++ b/src/Student.java	
@@ -0,0 +1,23 @@
+public class Student {
+
+    public String imie;
+    public String nazwisko;
+    public String nick;
+    public String email;
+    public String numerIndeksu;
+
+    public void przedstawSie() {
+        System.out.println("Nazywam sie" + imie +" " + nazwisko);
+
+    }
+    public void zalogujSie() {
+        System.out.println("Loguje sie za pomoca" + nick);
+    }
+    public void podajNrIndeksu() {
+        System.out.println("Mój numer indeksu to : " + numerIndeksu);
+    }
+
+    public void podajEmail() {
+        System.out.println("Mój email to : " + email);
+    }
+}
Index: src/Auto.java
===================================================================
diff --git a/src/Auto.java b/src/Auto.java
new file mode 100644
--- /dev/null	
+++ b/src/Auto.java	
@@ -0,0 +1,24 @@
+public class Auto {
+
+    public String marka;
+    public String model;
+    public int rok;
+    public int przebieg;
+
+    public void jedz() {
+        System.out.println("Jade!");
+
+    }
+
+    public void hamuj() {
+        System.out.println("Hamuje!");
+    }
+
+    public void info() {
+        System.out.println("Marka" + marka);
+        System.out.println("Model" + model);
+        System.out.println("Rok" + rok);
+        System.out.println("Przebieg" + przebieg);
+
+    }
+}
Index: src/AutoTest.java
===================================================================
diff --git a/src/AutoTest.java b/src/AutoTest.java
new file mode 100644
--- /dev/null	
+++ b/src/AutoTest.java	
@@ -0,0 +1,26 @@
+public class AutoTest {
+
+    public static void main(String[] args) {
+
+        Auto mercedes = new Auto();
+        mercedes.marka = "Mercedes";
+        mercedes.model = "Klasa S";
+        mercedes.rok =  2021;
+        mercedes.przebieg =  1000;
+
+        mercedes.jedz();
+        mercedes.hamuj();
+        mercedes.info();
+
+        Auto audi = new Auto();
+        audi.marka = "Audi";
+        audi.model = "A5";
+        audi.rok = 2021;
+        audi.przebieg = 0;
+
+        audi.jedz();
+        audi.hamuj();
+        audi.info();
+    }
+
+}
